components/server/src/jobs/org-only-migration-job.ts

```diff
@@ -33,8 +33,9 @@ export class OrgOnlyMigrationJob implements Job {
 
             const result: User[] = [];
             for (const user of users) {
-                result.push(await this.migrationService.migrateUser(user));
+                result.push(await this.migrationService.migrateUser(user, true, this.name));
```

Comment:
Before the 2nd argument was `false`, right?

Reply:
yes, but it should be true